postcss.config.js

```diff
@@ -4,5 +4,12 @@ const cssnano = require('cssnano');
 const foutWithAClass = require('postcss-fout-with-a-class').default;
 
 module.exports = {
-    plugins: [autoprefixer(), foutWithAClass({ families: ['Poppins', 'Roboto'], className: 'fonts-loaded' }), cssnano()]
+    plugins: [
+        autoprefixer(),
+        foutWithAClass({
+            families: ['Poppins', 'Roboto', 'caecilia', 'caecilia-sans-text'],
```

Comment:
Checked and this list works for both stylesheets, i.e. it's OK to have a single list here.